test/expr/unary/selector/selector.swift

```diff
@@ -116,7 +116,7 @@ func testParseErrors3(_ c1: C1) {
 
 func testParseErrors4() {
   // Subscripts
-  _ = #selector(C1.subscript) // expected-error{{type 'C1.Type' has no subscript members}}
+  _ = #selector(C1.subscript) // expected-error{{type 'C1' has no member 'subscript'}}
```

Comment:
I'm concerned about the *diagnostic.* It implies that the compiler is interpreting this syntax as a reference to a member named 'subscript', which would lead to a user thinking they've done something wrong, rather than this simply being a feature that's not implemented yet.

It's not a deal-breaker, but I consider us not supporting subscript selectors to just be a bug, and therefore the compiler shouldn't tell the user they've done something wrong.